Competition2018/src/main/java/competition/subsystems/elevator/commands/MoveToMaxHeightCommand.java

```diff
@@ -26,18 +24,21 @@ public MoveToMaxHeightCommand(ElevatorSubsystem elevator, PIDFactory pf) {
     @Override
     public void initialize() {
         log.info("Initializing");
+        if (!elevator.isCalibrated()) {
+            log.warn("ELEVATOR UNCALIBRATED - THIS COMMAND WILL NOT DO ANYTHING!");
+        }
     }
 
     @Override
     public void execute() {
-        double power = pid.calculate(max, elevator.getCurrentHeight());
-
-        elevator.setPower(power);
+        if (elevator.isCalibrated()) {
+            double power = pid.calculate(elevator.getMaxHeight(), elevator.getCurrentHeightInInches());
```

Comment:
Why does this command do the same thing as the other one? Shouldn't it just set the goal which the subsystem stores?

Reply:
I'm not sure what you're asking here, could you clarify?

Reply:
Agreed, these can be removed now.